util/file_util.py

```diff
@@ -428,9 +432,12 @@ def file_get_name(file_path: str,
   Returns:
     file name combined from path, suffix and extension.
   """
-    # Create the file directory if it doesn't exist.
+    if not file_path:
+        return ''
```

Comment:
Should we return error for blank file paths rather than empty? It would be easy to debug if empty path is the culprit

Reply:
by 'return error' do you mean `return None` or throw an exception? 

Throwing an exception complicates caller who has to catch the error rather than check for returned value.

the `if` check for return type works with both  `None and ``. Is the convention to return None rather than empty? The code may not be consistent in this as it uses a simple if check.

Reply:
I meant throwing an exception. Usually, for precondition/validation failures, ValueError is thrown by many python libraries. Indicating error via some code( returning blank) vs exceptions have their pros and cons. I do not see there being a convention. 

Betwee n`` and None, None seems more intuitive to me.

Reply:
Changed to None. The callers in rest of the code may not handle exceptions, can update this to more consistent error in future.